src/parkapi_sources/converters/opendata_swiss/converter.py

```diff
@@ -0,0 +1,79 @@
+from parkapi_sources.exceptions import ImportParkingSiteException, ImportSourceException
+from parkapi_sources.models import RealtimeParkingSiteInput, SourceInfo, StaticParkingSiteInput
+
+from .models import OpenDataSwissFeatureInput
+
+
+class OpenDataSwissBasePullConverter(PullConverter, ABC):
+    geojson_validator = DataclassValidator(GeojsonInput)
+    opendata_swiss_feature_validator: DataclassValidator
+
+    def _get_feature_inputs(self) -> tuple[list[OpenDataSwissFeatureInput], list[ImportParkingSiteException]]:
+        feature_inputs: list[OpenDataSwissFeatureInput] = []
+        import_parking_site_exceptions: list[ImportParkingSiteException] = []
+
+        response = requests.get(self.source_info.source_url, timeout=30)
+        response_data = response.json()
+
+        try:
+            geojson_input = self.geojson_validator.validate(response_data)
+        except ValidationError as e:
+            raise ImportSourceException(
+                source_uid=self.source_info.uid,
+                message=f'Invalid Input at source {self.source_info.uid}: {e.to_dict()}, data: {response_data}',
+            ) from e
+
+        for feature_dict in geojson_input.features:
+            try:
+                feature_input = self.opendata_swiss_feature_validator.validate(feature_dict)
+            except ValidationError as e:
+                import_parking_site_exceptions.append(
+                    ImportParkingSiteException(
+                        source_uid=self.source_info.uid,
+                        parking_site_uid=feature_dict.get('id'),
+                        message=f'Invalid data at uid {feature_dict.get("id")}: ' f'{e.to_dict()}, data: {feature_dict}',
+                    ),
+                )
+                continue
+
+            feature_inputs.append(feature_input)
+
+        return feature_inputs, import_parking_site_exceptions
+
+    def get_static_parking_sites(self) -> tuple[list[StaticParkingSiteInput], list[ImportParkingSiteException]]:
+        feature_inputs, import_parking_site_exceptions = self._get_feature_inputs()
+
+        static_parking_site_inputs: list[StaticParkingSiteInput] = []
+        for feature_input in feature_inputs:
+            static_parking_site_inputs.append(feature_input.to_static_parking_site_input())
+
+        return static_parking_site_inputs, import_parking_site_exceptions
+
+
+class OpenDataSwissPullConverter(OpenDataSwissBasePullConverter):
+    opendata_swiss_feature_validator = DataclassValidator(OpenDataSwissFeatureInput)
+
+    source_info = SourceInfo(
+        uid='opendata_swiss',
+        name='Open-Data-Plattform öV Schweiz (opentransport.swiss)',
+        public_url='https://opentransportdata.swiss/de/dataset/parking-facilities',
+        source_url='https://opentransportdata.swiss/dataset/c1be030e-adad-41c6-8ee2-899e85f840f6/resource/5dab3aba-6b5a-42cb-bb73-103fb28c7e10/download/parking-facilities.json',
```

Comment:
This `source_url` ends up in an HTTP 500. If I understand it correctly, the permalink is `https://opentransportdata.swiss/de/dataset/parking-facilities/permalink` instead, and you have to make sure that you follow the HTTP 302.

Reply:
Without the permalink, you don't get any data. Just try to access the url now :)

It seems that Python requests automatically follows HTTP 302s, just checked it using a Python interpreter.